### ISSUE TITLE
can't install cyg-get

### ISSUE THREAD

**Issue by thecotne** [chocolatey/chocolatey-coreteampackages#67]
```powershell
PS C:\> choco install cyg-get
Chocolatey v0.9.9.4
Installing the following packages:
cyg-get
By installing you accept licenses for the packages.

cyg-get v1.1.2
 Found 'chocolateyinstall.ps1':

$path = Join-Path $(Split-Path -parent $MyInvocation.MyCommand.Definition) 'cyg-get.ps1'

Install-ChocolateyPowershellCommand 'cyg-get' $path


Do you want to run the script?
 NOTE: If you choose not to run the script, the installation will
 fail.
 Skip is an advanced option and most likely will never be wanted.

 1) yes
 2) no [Default - Press Enter]
 3) skip
1
 WARNING: Some imported command names include unapproved verbs which might make
 them less discoverable.  Use the Verbose parameter for more detail or type
 Get-Verb to see the list of approved verbs.
 Adding C:\ProgramData\bin\cyg-get.bat and pointing it to powershell command C:\ProgramData\chocolatey\lib\cyg-get\tools
\cyg-get.ps1
 Out-File : Could not find a part of the path 'C:\ProgramData\bin\cyg-get.bat'.
 At C:\ProgramData\chocolatey\helpers\functions\Install-ChocolateyPowershellComm
 and.ps1:40 char:102
 + powershell -NoProfile -ExecutionPolicy unrestricted -Command ""& `'$psFileFul
 lPath`'  %*"""| Out-File <<<<  $packageBatchFileName -encoding ASCII
     + CategoryInfo          : OpenError: (:) [Out-File], DirectoryNotFoundExce
    ption
     + FullyQualifiedErrorId : FileOpenFailure,Microsoft.PowerShell.Commands.Ou
    tFileCommand
cyg-get install not successful.
Error while running 'C:\ProgramData\chocolatey\lib\cyg-get\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 package(s). 1 package(s) failed.
 See the log for details.
Failures:
 - cyg-get
```

**Comment by gep13** [comment]
@ferventcoder can you add anything to this issue?

**Comment by ferventcoder** [comment]
This doesn't appear to be an issue with the latest version of choco. The use of `$nugetExePath` is what did it. It was fixed by https://github.com/chocolatey/choco/issues/267

**Comment by ferventcoder** [comment]
However it should have not affected 0.9.9.4 - https://github.com/chocolatey/choco/commit/cb3aed93

**Comment by ferventcoder** [comment]
Found it! https://github.com/chocolatey/choco/commit/a830a7095d1529582755a0460ef28a72b2dcdde6

Fixed in 0.9.9.5

**Comment by JulianIsrael** [comment]
Is this really fix? I has having same issue in latest version cyg-get 1.2.1 what i did was downloaded the setup-x86_64.exe from https://cygwin.com/setup-x86_64.exe put it n c:\toos\cygwin renamed with cygwinsetup.exe and it worked.
